resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/PVTestsSuite.scala

```diff
@@ -156,12 +167,18 @@ private[spark] trait PVTestsSuite { k8sSuite: KubernetesSuite =>
 }
 
 private[spark] object PVTestsSuite {
-  val STORAGE_NAME = "test-local-storage"
+  val STORAGE_NAME = "test-local-storage-2"
```

Comment:
So, is this to avoid the Jenkins pre-configured storage?

Reply:
This was just to make sure I had got all of the places we used the raw string. I'll switch this back.